site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -1428,3 +1428,71 @@ for more information about RPC and HTTP service names, and how to use it.
 </Tip>
 
 [Publisher]: https://www.reactive-streams.org/reactive-streams-1.0.2-javadoc/org/reactivestreams/Publisher.html
+
+## Kotlin Coroutines Support
+
+You can implement annotated services with Kotlin Coroutines.
+
+```kotlin
+class MyAnnotatedService {
+    @Get("/users/{name}")
+    suspend fun getUserToken(@Param("name") name: String): HttpResponse {
+        val user = myReactiveRepository.findByName(name).await()
+        return HttpResponse.of(user.token)
+    }
+}
+```
+
+By default, armeria uses a context-aware event loop as a [coroutine dispatcher][CoroutineDispatcher].
+If you annotates methods with `@Blocking` or enables `useBlockingTaskExecutor`,
+a context-aware blocking executor is used as a dispatcher.
+
+```kotlin
+serverBuilder
+    .annotatedService()
+    // enables `useBlockingTaskExecutor`
+    .useBlockingTaskExecutor(true)
+    .build(MyAnnotatedService())
+
+// or
+
+import com.linecorp.armeria.server.annotation.Blocking
+
+class MyAnnotatedService {
+    @Blocking
+    @Get("/blocking/users/{name}")
+    suspend fun getUserTokenBlocking(@Param("name") name: String): HttpResponse {
+        val id = myReactiveClient.getId(name).await()
+        val user = myBlockingRepository.findById(id)
+        return HttpResponse.of(user.token)
+    }
+}
+```
+
+<Tip>
+
+If you use [Logback integration][logback] and/or [Zipkin integration][zipkin],
+coroutines should be executed in threads context aware of [RequestContext][ctx].
```

Comment:
`... executed in a <type://RequestContext>-aware thread.` ?

If `>-aware` breaks a link, then `... executed in a thread that is aware of <type://RequestContext>.`

Reply:
I rewrite to the latter one.